autozone-poc/src/components/Shops/Catagory/Catagory.test.js

```diff
@@ -0,0 +1,44 @@
+import '@testing-library/jest-dom/extend-expect';
+import renderer from "react-test-renderer";
+import React, { Component } from 'react';
+import { configure, mount, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { render } from '@testing-library/react';
+import Catagory from './Catagory';
+
+configure({ adapter: new Adapter() });
+
+
+describe('Catagory testing', () => {
+    var categoryList = [{
```

Comment:
instead of creating array, pass data directly. and check for different cases

Reply:
This page is not associated to me.